02_bubble/selva/bubblesort.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(items []int) []int {
+
+	if len(items) > 1 {
+		pivotIndex := len(items) / 2
+		var smallerItems = []int{}
+		var largerItems = []int{}
+
+		for i := range items {
```

Comment:
Something about this implementation seems a bit more verbose than necessary - look up reference implmentations please.